### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request is about a react project called the capstone project. In this project, we :
- Use linters, React, Redux as tech
- We create a mobile version
- We fetch an API
- We make a filter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,150 +2,182 @@
...
+- **Demo of Linters**
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 💻 Getting Started <a name="getting-started"></a>
 
-To get a local copy up and running, follow these steps.
+🚀 Live Demo Link (https://react-capstone-ot6v.onrender.com/)
+
```
Comment (t-yanick) [inline README.md:59]: - [ ] Great job working on this capstone project. As per project requirements, you are supposed to do a video presentation of your project. Kindly do that and provide the link in the re-review request.
Screen Shot 2023-09-09 at 6 28 26 AM

File: src/redux/stats/statsSlice.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchStatsAsync.fulfilled, (state, action) => {
+        const newStats = [];
+        const keys = Object.keys(action.payload);
+        keys.forEach((keyOfActionPayload) => {
+          newStats.push({
+            id: keyOfActionPayload,
+            ...action.payload[keyOfActionPayload],
```
Comment (t-yanick) [inline src/redux/stats/statsSlice.js:31]: - [ ] As per project requirements you are to use a `Filter stateless component `to `` data retrieved from the API. But I don't see you using the filter method anywhere.
Screen Shot 2023-09-09 at 7 01 49 AM

File: src/__tests__/App.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import store from '../Redux/store';
+import App from '../App';
+
```
Comment (t-yanick) [inline src/__tests__/App.test.js:7]: - [ ] Kindly add some unit tests for the app's components and some pure functions in the Redux code

File: src/components/StatAll.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+              <p>{stat.calendarYear}</p>
+              <h3>{stat.symbol}</h3>
+              <h2>
+                $
+                { shortRevenue(stat.revenue)}
+                {' '}
+                bills
+              </h2>
+            </div>
+          </Link>
```
Comment (t-yanick) [inline src/components/StatAll.js:75]: - [ ] As per the given design, the full details section also has two columns and the details are to the right. See the screenshots below and fix this. Your design has just one column and the details to the left.
Your design
Screen Shot 2023-09-09 at 7 15 42 AM

Given design
Screen Shot 2023-09-09 at 7 16 25 AM

File: src/__tests__/App.test.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+describe('App js testing', () => {
```
Comment (t-yanick) [inline src/__tests__/App.test.js:8]: - [ ] Your test cases are all failing. Kindly review and fix them.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @MenendezON , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App deployed using Render
:heavy_check_mark: Has basic project structure
:heavy_check_mark: The project contains a `Home` page and a `Detail` page
:heavy_check_mark: The project is a single page application (SPA) using `react-router-dom` to serve each page without reloading.
:heavy_check_mark: The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Kindly check your console to ensure that there are no errors.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/App.test.js (hunk 5 above)
Comment (skyv26) [inline src/__tests__/App.test.js:8]: - [ ] As your previous code reviewer mentioned about the test failing, it seems like the problem still persists and needs action to solve it.

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @MenendezON, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/App.test.js (hunk 5 above)
Comment (MenendezON) [reply src/__tests__/App.test.js:8]: I don't know why is not working on your computer but it works on mine.
Please @skyv26, take a look at this short video.
https://www.loom.com/share/4c7b5681fe864c5081a7dd15b1dc3f84?sid=95d9841b-5788-473b-9600-e8195ca39253

Comment (skyv26) [reply src/__tests__/App.test.js:8]: @MenendezON Please delete your `__snapshots__` folder and try it again and if everything seems fine re-submit your project again.

Comment (peterrobert) [reply src/__tests__/App.test.js:8]: this change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @MenendezON, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work on your project. Awesome work making the changes that were requested by the previous reviewer. ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**